src/_tests-/home.test.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { createStore, combineReducers } from 'redux';
+import Home from '../containers/Home';
+import filterReducer from '../redux/reducers/filter';
```

Comment:
for your tests, your app should have the following
> Integration test using react testing libraries 
- you should perform a mocking data fetch
-  you should perform a Redux store connection test 😄  👋🏾

Reply:
I have done according to the instruction

-    You may need to mock the access to the API, so that your tests don't send actual requests.

![image](https://user-images.githubusercontent.com/65109017/194122979-022e52d0-b8c1-4b9b-95b7-908e4e64126b.png)

- You may need to mock the connection to the Redux Store.

![image](https://user-images.githubusercontent.com/65109017/194123304-8a47899b-a0b1-444e-9d69-48330c4cf4c8.png)